lms/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## 0.1.0 (2024-01-30)
+- Initial release
```

Comment:
Leave this blank for now (the date is already invalid) and we can bump it with an actual release:

```suggestion
```